markdown/developers/adding_pipelines.md

```diff
@@ -299,6 +297,22 @@ Basic rules for such contributions:
 Sometimes, especially when adding new features to a pipeline, the dependencies change as well. In such cases, you might want to have an updated Docker Container available before submitting a pull request, in order to have the GitHub Actions tests run through when testing your updated code. To achieve that, please follow these steps:
 
 - Add _only_ the newly required dependencies to the `environment.yml` in the pipeline code
+- If you only add new processes to an already existing pipeline however, you can simply specify the container in the `nextflow.config` file, like so:
+
+```bash
+process {
+    withName:foo {
+        container = 'image_name_1'
+    }
+    withName:bar {
+        container = 'image_name_2'
+    }
+    }
+charliecloud {
+    enabled = true
+}
+```
```

Comment:
I think prettier wants it to have a space

Reply:
I swear to god prettier is giving me PTSD, i gotta bring it up with my therapist after this hackaton